packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -81,6 +109,13 @@ export default createFragmentContainer(RetroTopic, {
     fragment RetroTopic_topic on RetroReflectionGroup {
       title
       voteCount
+      thread(first: 101) @connection(key: "DiscussionThread_thread") {
```

Comment:
ooo, this is gonna get pretty expensive for the DB, maybe instead we just have a "commentCount" field? it returns the `count` on the comment table indexed on the thread id.

this fixes the thread/comment ambiguity. for example with threads, if we say "see 10 comments" and all 10 are tasks, there won't be any comments.

Reply:
@mattkrick perfect. I’m looking at adding a `commentCount` field to the type using `await dataLoader.get('commentsByThreadId').load(reflectionGroupId)` but now that I think about it that may be expensive, too? Instead should I add a commentCount field to the database record for the group and update that count with mutations?